generators/app/templates/src/environments/environment.ts

```diff
@@ -3,9 +3,12 @@
 // `ng build --env=prod` then `environment.prod.ts` will be used instead.
 // The list of which env maps to which file can be found in `.angular-cli.json`.
 
+// `.env.ts` is generated by the `npm run env` command
+import env from './.env';
+
 export const environment = {
   production: false,
-  version: '(dev)',
+  version: env.npm_package_version + '(dev)',
```

Comment:
Can you use `env.npm_package_version + '-dev',` so the version is semver compatible then?

Reply:
ok.